app/views/achievements/index.html.haml

```diff
@@ -1,4 +1,4 @@
 .row
-  .col-xs-12
+  .col-md-12
```

Comment:
Cool. What clients is the site designed for? I'm only using it on a laptop/desktop.

I'll back out my change in any case.